src/App.vue

```diff
@@ -37,7 +37,7 @@
 			</div>
 		</Modal>
 
-		<router-view v-show="!cardDetailsInModal || !$route.params.cardId" name="sidebar" />
+		<router-view name="sidebar" :visible="!cardDetailsInModal || !$route.params.cardId" />
```

Comment:
Can't use v-if directly on the router-view?

Reply:
There seems to be some issue with the nested router views that are used for sub-routes, where the router tried to render the component to a non-existing router-view element then. That is why I went that way for now. But I have that on my list to investigate further, so this is more a temporary workaround.